### DISCUSSION TITLE
exited with code=3221225781 in VS code

### DISCUSSION THREAD

**Discussion by FatemehEnsaf** [prisma/prisma#12428] [Q&A]
Guys, when I run, I see these in my output section. Can someone tell me what should I do?
" [Running] cd "c:\Users\Lenovo\AppData\Local\Temp\Rar$DIa1580.44752\" && gcc hello0.c -o hello0 && "c:\Users\Lenovo\AppData\Local\Temp\Rar$DIa1580.44752\"hello0

[Done] exited with code=3221225781 in 0.449 seconds "

**Comment by janpio** [comment]
When you run what exactly? What is the relation to Prisma?

**Reply by vladi-stevanovic** [reply]
👋 Hello @FatemehEnsaf  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
